be/src/olap/snapshot_manager.cpp

```diff
@@ -546,4 +576,34 @@ OLAPStatus SnapshotManager::_create_snapshot_files(
     return res;
 }
 
+OLAPStatus SnapshotManager::_convert_beta_rowsets_to_alpha(const TabletMetaSharedPtr& new_tablet_meta,
+                const vector<RowsetMetaSharedPtr>& beta_rowsets, const std::string& dst_path,
+                std::vector<RowsetMetaSharedPtr>* new_rowsets, bool* modified) {
+    OLAPStatus res = OLAP_SUCCESS;
+    RowsetConverter rowset_converter(new_tablet_meta);
+    for (auto& rowset_meta : beta_rowsets) {
+        if (rowset_meta->rowset_type() == BETA_ROWSET) {
+            *modified = true;
+            RowsetMetaPB rowset_meta_pb;
+            LOG(INFO) << "convert beta rowset:" << rowset_meta->rowset_id() << " to alpha";
```

Comment:
Do we need this log? If we leave a log, this log lack some key information, such as tablet_id, schema_hash.

Reply:
I add this log to record which rowset is converted. I add the converted rowset id to the log.
Tablet id can be determined in the context.